index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>DOM Manipulations 2</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <!-- 1) Create a counter similar to the one included in the screenshot below / in the presentation.
+    - There are two buttons, a plus and a minus one. Initially, the plus button adds 1 to the counter, the minus button subtracts 1 from the number. 
+    - The counter displays the current sum. By default it should start as 0.  -->
+    <h2>Counter 1</h2>
+    <div>
+        <button class="button" id="minus">-</button>
+        <span class="field" id="counter">0</span>
+        <button class="button" id="plus">+</button>
+    </div>
+
+
+    <!-- 2) Adjust the minus button to not allow the counter to go below 0 (only allow subtracting if the current count is positive).  -->
+    <h2>Counter 2</h2>
+    <div>
+        <button id="minus2">-</button>
+        <span class="field" id="counter2">0</span>
+        <button id="plus2">+</button>
+    </div>
+
+
+    <!-- Add an additional button that checks if the current number in the counter is odd or even.
+    - If the number is even, change the counter color to green.
+    - If the number is odd, change the counter color to red.
+    - Reset the counter color to black if the user clicks on the plus or the minus buttons.  -->
+
+    <h2>Counter 3</h2>
+    <div>
+        <button id="minus3">-</button>
+        <span class="field" id="counter3">0</span>
+        <button id="plus3">+</button>
+        <button id="isEven">Is even?</button>
+    </div>
+
+<!-- Add an additional input with a default value of 1
+    - Allow the user to input any number between 0 and 10. 
+    - Once the user enters the value, the plus and minus buttons should add/subtract this value instead of subtracting 1
+    - it's okay to assume the user is friendly and will not input anything else then integers into the input
+    - (optional) Adjust the check from the second exercise to make sure the user cannot cheat and for example go from 3 to -1 by changing the input to 4 and clicking the minus button.  -->
+
+    <h2>Counter 4</h2>
+    <div>
+        <button id="minus4">-</button>
+        <span class="field" id="counter4">0</span>
+        <button id="plus4">+</button>
+        <button id="isEven4">Is even?</button>
+        <input id="user-input" type="number" value="1" max="10" min="0">
```

Comment:
It's generally a good practice to keep one type of casing for the same items throughout the code, `isEven4` uses the camelCase and here the id is in kebab-case. But that's minor!